Helper/FeatureSwitch/Definitions.php

```diff
@@ -434,5 +439,11 @@ class Definitions
             self::DEFAULT_VAL_KEY => false,
             self::ROLLOUT_KEY     => 0
         ],
+        self::M2_PREVENT_SETTING_BOLT_IPS_AS_CUSTOMER_IP_ON_QUOTE => [
+            self::NAME_KEY        => self::M2_PREVENT_SETTING_BOLT_IPS_AS_CUSTOMER_IP_ON_QUOTE,
+            self::VAL_KEY         => true,
+            self::DEFAULT_VAL_KEY => false,
+            self::ROLLOUT_KEY     => 0
```

Comment:
Why do you want to disable it by default?

Reply:
Good point. Will switch to enabled.

Reply:
```suggestion
            self::ROLLOUT_KEY     => 100
```